spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/IntegerEncoderDecoder.java

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
should always be one space after the header, before the package declaration

can you please regenerate this PR? (also we've had some additions with more violators since this was submitted)